tst/EnergyPlus/unit/PurchasedAirManager.unit.cc

```diff
@@ -473,7 +478,10 @@ TEST_F(ZoneIdealLoadsTest, IdealLoads_ExhaustNodeTest)
         "  ,                               !- Outdoor Air Economizer Type",
         "  ,                               !- Heat Recovery Type",
         "  ,                               !- Sensible Heat Recovery Effectiveness{ dimensionless }",
-        "  ;                               !- Latent Heat Recovery Effectiveness{ dimensionless }",
+        "  ,                               !- Latent Heat Recovery Effectiveness{ dimensionless }",
+        "  ,                               !- Design Specification ZoneHVAC Sizing Object Name }",
+        "  DXHeatingCoilFuelEffSched,      !- Heating Fuel Efficiency Schedule Name }",
+        "  DXCoolingCoilFuelEffSched;      !- Cooling Fuel Efficiency Schedule Name }",
```

Comment:
This unit test has been modified to demonstrate that the scheduled fuel efficiency inputs are processed correctly.